webpackBaseConfig.js

```diff
@@ -8,24 +9,7 @@ module.exports = {
   context: path.resolve(__dirname),
   resolve: {
     modules: [__dirname, 'node_modules'],
-    alias: {
-      '@mui/markdown': path.resolve(__dirname, './packages/markdown'),
-      '@mui/material': path.resolve(__dirname, './packages/mui-material/src'),
-      '@mui/docs': path.resolve(__dirname, './packages/mui-docs/src'),
-      '@mui/icons-material': path.resolve(__dirname, './packages/mui-icons-material/lib/esm'),
-      '@mui/lab': path.resolve(__dirname, './packages/mui-lab/src'),
-      '@mui/styled-engine': path.resolve(__dirname, './packages/mui-styled-engine/src'),
-      '@mui/styled-engine-sc': path.resolve(__dirname, './packages/mui-styled-engine-sc/src'),
-      '@mui/styles': path.resolve(__dirname, './packages/mui-styles/src'),
-      '@mui/system': path.resolve(__dirname, './packages/mui-system/src'),
-      '@mui/private-theming': path.resolve(__dirname, './packages/mui-private-theming/src'),
-      '@mui/base': path.resolve(__dirname, './packages/mui-base/src'),
-      '@mui/utils': path.resolve(__dirname, './packages/mui-utils/src'),
-      '@mui/material-next': path.resolve(__dirname, './packages/mui-material-next/src'),
-      '@mui/joy': path.resolve(__dirname, './packages/mui-joy/src'),
-      'typescript-to-proptypes': path.resolve(__dirname, './packages/typescript-to-proptypes/src'),
-      docs: path.resolve(__dirname, './docs'),
-    },
+    alias: getMuiAliases({ type: 'src', useESIcons: true }),
```

Comment:
One thing that could be worth a try is [reading them from tsconfig](https://www.npmjs.com/package/tsconfig-paths-webpack-plugin)

Reply:
If we can drop the babel aliases, I guess we could have the tsconfig as the single source of truth.
Would you be ok to keep it for a follow up?

Reply:
Yes absolutely